include/albatross/src/core/linear_combination.hpp

```diff
@@ -74,6 +76,16 @@ inline auto difference(const X &x, const Y &y) {
   return difference(vx, vy);
 }
 
+template <typename X> inline auto to_linear_combination(const X &x) {
+  return LinearCombination<X>({x}, Eigen::VectorXd::Ones(1));
+}
+
+template <typename X>
+inline auto to_linear_combination(const LinearCombination<X> &x) {
```

Comment:
What is the point of this function?

Reply:
If you aren't sure what the type of `X` is it could be handy to be able to call
```
Foo foo;
LinearCombination<Foo> a = to_linear_combination(foo);
```
and
```
LinearCombination<Foo> already_a_combo;
LinearCombination<Foo> b = to_linear_combination(already_a_combo);
```
without the second returning:
```
LinearCombination<LinearCombination<Foo>>
```

Reply:
Might this be worth a comment clarifying it for the next reader?  (Possibly me in a year?)